python/mlcroissant/mlcroissant/_src/operation_graph/operations/read.py

```diff
@@ -89,6 +96,13 @@ def _read_file_content(self, encoding_format: str, file: Path) -> pd.DataFrame:
             download_git_lfs_file(file)
         reading_method = _reading_method(self.node, self.fields)
 
+        if encoding_format == EncodingFormat.ARFF:
+            if scipy is None:
```

Comment:
You shouldn't need this as this is already checked for when lazily loading ([source](https://github.com/mlcommons/croissant/blob/5a2e5201ae23d83c992948e7a960dad95f5ea7f3/python/mlcroissant/mlcroissant/_src/core/optional.py#L24-L28)).

Reply:
I need if for pytype :S I'm doing the same as https://github.com/mlcommons/croissant/blame/5a2e5201ae23d83c992948e7a960dad95f5ea7f3/python/mlcroissant/mlcroissant/_src/torch/torch_adapter/dataloader.py#L72